cf/data/mixin/deprecations.py

```diff
@@ -121,22 +129,99 @@ def in_memory(self):
             removed_at="5.0.0",
         )  # pragma: no cover
 
-    def files(self):
-        """Deprecated at version 3.4.0, use method `get_filenames`
-        instead."""
-        _DEPRECATION_ERROR_METHOD(
+    @property
+    def ismasked(self):
+        """True if the data array has any masked values.
+
+        Deprecated at version TODODASK. Use the `is_masked` attribute
+        instead.
+
+        **Examples**
+
+        >>> d = cf.Data([[1, 2, 3], [4, 5, 6]])
+        >>> print(d.ismasked)
+        False
+        >>> d[0, ...] = cf.masked
+        >>> d.ismasked
+        True
+
+        """
+        _DEPRECATION_ERROR_ATTRIBUTE(
             self,
-            "files",
-            "Use method `get_filenames` instead.",
-            version="3.4.0",
+            "ismasked",
+            message="Use the 'is_masked' attribute instead",
+            version="TODODASK",
+            removed_at="5.0.0",
+        )  # pragma: no cover
+
+    @property
+    def ispartitioned(self):
+        """True if the data array is partitioned.
+
+        **Examples**
+
+        >>> d._pmsize
+        1
+        >>> d.ispartitioned
+        False
+
+        >>> d._pmsize
+        2
+        >>> d.ispartitioned
+        False
+
+        """
+        _DEPRECATION_ERROR_ATTRIBUTE(
+            self, "ispartitioned", version="TODODASK", removed_at="5.0.0"
         )  # pragma: no cover
 
     @property
     def unsafe_array(self):
-        """Deprecated at version 3.0.0, use `array` attribute
-        instead."""
+        """Deprecated at version 3.0.0.
+
+        Use the `array` attribute instead.
+
+        """
+        _DEPRECATION_ERROR_ATTRIBUTE(
+            self,
+            "unsafe_array",
+            message="Use the 'array' attribute instead.",
+            version="3.0.0",
+            removed_at="4.0.0",
+        )  # pragma: no cover
+
+    @property
+    def varray(self):
+        """A numpy array view the data array.
```

Comment:
```suggestion
        """A numpy array view of the data array.
```

(Not an issue from the shift over to the deprecations module, but let's add in the missing word?)